src/tests/ftest/nvme/nvme_io_verification.py

```diff
@@ -106,3 +106,84 @@ def test_nvme_io_verification(self):
 
             # destroy pool
             self.destroy_pools(self.pool)
+
+    @avocado.fail_on(DaosApiError)
+    def test_nvme_server_restart(self):
+        """Jira ID: DAOS-2650.
+
+        Test Description:
+            Test will run IOR with non standard transfer sizes for different
+            set of pool sizes. Purpose is to verify io transaction to scm and
+            nvme for different pool sizes when servers are restarted after
+            write.
+
+        Use Cases:
+            (1) Running IOR with different set of transfer size where first
+            transfer size is < 4096 and then > 4096. Verify the data after
+            servers are restarted.
+            (2) Repeat the case(1) with maximum nvme pool size that can be
```

Comment:
Pool is getting created with large size but how much % of pool is getting filled up? I see block_size: 20000000 x 16 process = 305MB. Can we increase the data set for single test case to make sure we are really filling the SCM/NVMe?

Reply:
Done